README.md

```diff
@@ -449,8 +449,8 @@ The following settings are optional and allow you to further configure your clus
   [default]
   region = us-west-2
   credential_process = aws_signing_helper credential-process \
-     --certificate /var/lib/kubelet/pki/kubelet-client-current.pem \
-     --private-key /var/lib/kubelet/pki/kubelet-client-current.pem \
+     --certificate /etc/kubernetes/pki/kubelet-server.crt \
+     --private-key /etc/kubernetes/pki/private/kubelet-server.key \
```

Comment:
Do we want to make this change? AFAIK `/var/lib/kubelet/pki/kubelet-client-current.pem` is the correct path for the default clusters created by EKS-A.

Reply:
Yeah, that's probably better to keep that. Reverted the change.